pycobertura/cobertura.py

```diff
@@ -114,27 +132,33 @@ def branch_rate(self, filename=None):
         else:
             el = self._get_class_element_by_filename(filename)
 
-        return float(el.attrib["branch-rate"])
+        return float(el.get("branch-rate"))
 
     @memoize
     def missed_statements(self, filename):
         """
         Return a list of uncovered line numbers for each of the missed
         statements found for the file `filename`.
         """
-        el = self._get_class_element_by_filename(filename)
-        lines = el.xpath("./lines/line[@hits=0]")
-        return [int(line.attrib["number"]) for line in lines]
+        classes = self._get_class_elements_by_filename(filename)
+        return [
+            int(line.get("number"))
+            for cl in classes
+            for line in cl.xpath("./lines/line[@hits=0]")
+        ]
```

Comment:
Same here, can we rename for consistency?

```suggestion
        classElements = self._get_class_elements_by_filename(filename)
        return [
            int(line.get("number"))
            for classElement in classElements
            for line in classElement.xpath("./lines/line[@hits=0]")
        ]
```

Reply:
done in https://github.com/aconrad/pycobertura/pull/132/commits/45e1b7b103bd59c9ff4756375a69fbcd479d0608